ApiFirst/AppServices/Validator/CustomerCreateDTOValidator.cs

```diff
@@ -0,0 +1,132 @@
+﻿using AppModels.AppModels;
+using FluentValidation;
+
+namespace AppServices.Validation
+{
+    public class CustomerCreateDTOValidator : AbstractValidator<CreateCustomerDTO>
+    {
+        public CustomerCreateDTOValidator()
+        {
+            RuleFor(x => x.Cpf).
+                NotEmpty().
+                NotEqual(" ").
+                Length(11).
+                Must(IsValidCpf).WithMessage("Please enter a valid CPF, this CPF is not valid");
+
+            RuleFor(x => x.Email).
+                EmailAddress().
+                Equal(x => x.EmailConfirmation).
+                NotEqual(" ").
+                MinimumLength(3).
+                MaximumLength(50).
+                NotEmpty();
```

Comment:
1. . vai na frente e não atrás
2. 
```suggestion
            RuleFor(x => x.Email).
                EmailAddress().
                Equal(x => x.EmailConfirmation).
                .NotEmpty()
                MinimumLength(12).
                MaximumLength(50);
```